stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -128,6 +128,7 @@ spec:
         - name: {{ template "mongodb-replicaset.name" . }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+{{ include "mongodb-replicaset.envVars" . | indent 10 }}
```

Comment:
I would do it with `toYaml`, that's more flexible. And call it `extraVars` instead. See also #4283.